website/content/en/docs/concepts/scheduling.md

```diff
@@ -152,6 +152,7 @@ Take care to ensure the label domains are correct. A well known label like `karp
 | karpenter.k8s.aws/instance-family                              | g4dn        | [AWS Specific] Instance types of similar properties but different resource quantities                                                                           |
 | karpenter.k8s.aws/instance-size                                | 8xlarge     | [AWS Specific] Instance types of similar resource quantities but different properties                                                                           |
 | karpenter.k8s.aws/instance-cpu                                 | 32          | [AWS Specific] Number of CPUs on the instance                                                                                                                   |
+| karpenter.k8s.aws/instance-cpu-manufacturer                    | aws          | [AWS Specific] Name of the CPU manufacturer                                                                                                                   |
```

Comment:
Can we remove this from `docs` folder? We only want this change in `preview` folder as it is not released yet.